resource-watch/resource-watch
# Implement update to effectively separate admin from user content in the back office

![image](https://user-images.githubusercontent.com/545342/71362177-9cd85700-2595-11ea-8821-66893e605da5.png)

## Overview
This PR adds a radio button to the back office that allows to load separately content from users with roles `ADMIN` and `USER`.

## Testing instructions
Go to the back office and check the tables for:
- dataset
- widget
- layer
- dashboards

## [Pivotal task](https://www.pivotaltracker.com/story/show/167344998)
